tests/baselines/reference/typedefNameless.errors.txt

```diff
@@ -0,0 +1,70 @@
+tests/cases/conformance/jsdoc/a.js(11,1): error TS2693: 'nameless' only refers to a type, but is being used as a value here.
+tests/cases/conformance/jsdoc/a.js(26,5): error TS2693: 'notOK' only refers to a type, but is being used as a value here.
+tests/cases/conformance/jsdoc/a.js(32,7): error TS1155: 'const' declarations must be initialized.
+
+
+==== tests/cases/conformance/jsdoc/a.js (3 errors) ====
+    /**
+     * @typedef {number}
+     */
+    var nameless;
+    
+    /**
+     * @typedef {number} named
+     */
+    var this_is_not_the_name = true;
+    
+    nameless = 123; // nameless is not a value
+    ~~~~~~~~

```

Comment:
I don't like this error. We can't change the semantics of JS or TS such that `nameless` no longer has a value meaning.